modules/apps/object/object-entry-folder-rest-impl/rest-openapi.yaml

```diff
@@ -1,8 +1,304 @@
+            operationId: deleteObjectEntryFolder
+            parameters:
+                - in: path
+                  name: objectEntryFolderId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+            responses:
+                204:
+                    content:
+                        application/json: {}
+                        application/xml: {}
+            tags: ["ObjectEntryFolder"]
+        get:
+            description:
+                Retrieves the object entry folder.
+            operationId: getObjectEntryFolder
+            parameters:
+                - in: path
+                  name: objectEntryFolderId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+                - in: query
+                  name: fields
+                  schema:
+                      type: string
+                - in: query
+                  name: nestedFields
+                  schema:
+                      type: string
+                - in: query
+                  name: restrictFields
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntryFolder"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntryFolder"
+            tags: ["ObjectEntryFolder"]
+        patch:
+            description:
+                Updates only the fields received in the request body, leaving any other fields untouched.
+            operationId: patchObjectEntryFolder
+            parameters:
+                - in: path
+                  name: objectEntryFolderId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/ObjectEntryFolder"
+                    application/xml:
+                        schema:
+                            $ref: "#/components/schemas/ObjectEntryFolder"
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntryFolder"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntryFolder"
+            tags: ["ObjectEntryFolder"]
+        put:
```

Comment:
This method should not be in this endpoint.

This is an upsert operation and we could not reuse the received objectEntryFolderId when adding it.

PUT method should only be in the /by-external-reference-code/ cases since we could pass it to the ADD methods